receptor_affinity/debugger.py

```diff
@@ -13,34 +13,65 @@
 
 
 class DiagNode:
+    """
+    A DiagNode takes a datapath, a node_id and a listener port and spins up an instance of a
```

Comment:
Can this docstring start by stating what a `DiagNode` is, before progressing to the parameters it takes? Something like:

```
"""A wrapper around a receptor process that's configured to provide debugging info.

Accepts the following parameters:

...
"""
```